core/runtime/src/main/java/io/quarkus/runtime/ApplicationConfig.java

```diff
@@ -22,4 +22,11 @@
      */
     @ConfigItem
     public Optional<String> version;
+
+    /**
+     * The header to use for UI Screen (Swagger UI, GraphQL UI etc)
+     * If not set, defaults to {applicationName} {applicationVersion} (powered by Quarkus {quarkusVersion}).
+     */
+    @ConfigItem
+    public Optional<String> uiheader;
```

Comment:
I think `uiHeader` would be better.

Reply:
Ok cool. so that will translate into ui-header in the config right ?